druid/examples/list.rs

```diff
@@ -34,31 +44,79 @@ fn main() {
 fn ui_builder() -> impl Widget<AppData> {
     let mut root = Flex::column();
 
+    // Build a button to add children to both lists
     root.add_child(
-        SizedBox::new(Button::new("Add", |_, data: &mut AppData, _| {
-            let mut d = (*data).deref().to_owned();
-            d.push((d.len() + 1) as u32);
-            *data = d.into();
-        }))
-        .height(30.0),
+        Button::new("Add", |_, data: &mut AppData, _| {
+            // Add child to left list
+            let value = data.left.len() + 1;
+            Arc::make_mut(&mut data.left).push(value as u32);
+
+            // Add child to right list
+            let value = data.right.len() + 1;
+            Arc::make_mut(&mut data.right).push(value as u32);
+        })
+        .fix_height(30.0),
         0.0,
     );
 
-    root.add_child(
+    let mut lists = Flex::row();
+
+    // Build a simple list
+    lists.add_child(
         Scroll::new(List::new(|| {
-            SizedBox::new(
-                Container::new(Padding::new(
-                    10.0,
-                    DynLabel::new(|d, _| format!("List item #{}", d)),
-                ))
-                .background(Color::rgb(0.5, 0.5, 0.5)),
-            )
-            .expand()
-            .height(50.0)
+            DynLabel::new(|item: &u32, _| format!("List item #{}", item))
+                .padding(10.0)
+                .expand()
+                .height(50.0)
```

Comment:
this is interesting; it's (maybe) a fluke that this works, because `height()` is not a fn on WidgetExt, but only on SizedBox; there's `fix_height(5.0)` on WidgetExt, but I don't love that name. Thinking about this a bit more, if the `WidgetExt` method was *also* called `height` this would probably just work? 🤔

Reply:
I think that we should unify the names. If we have `width` and `height` on `SizedBox` then we should have the same names on `WidgetExt` or vice versa. 
What do you think?

Reply:
yea, I think that makes sense, I think I hesitated because those two felt names overly generic, but I agree that would be an improvement.